spec/lib/schema_file_loader_spec.rb

```diff
@@ -14,7 +14,7 @@
     ]
   end
 
-  context "#s3_credentials" do
+  describe "#s3_credentials" do
```

Comment:
rubocop yelled at me that this is a description of a method, not a context & I agreed